js/main.js

```diff
@@ -21,6 +25,15 @@ function getRandomInt(max) {
   return Math.floor(Math.random() * max);
 }
 
+function submitSearch(event) {
+  event.preventDefault();
+  var mobileSearch = $mobileSearchContent.value;
+  var desktopSearch = $desktopSearchContent.value;
+  if (mobileSearch || desktopSearch !== '') {
+    return mobileSearch || desktopSearch;
```

Comment:
Event handlers do not have return statements, since you can't access the return.